test/test_jit.py

```diff
@@ -10551,6 +10551,76 @@ def addmm_grad_test(b, x, w):
         self.assertEqual(w.grad, w_ref.grad)
         self.assertEqual(b.grad, b_ref.grad)
 
+    def test_layer_norm_grad(self):
```

Comment:
`with enable_profiling_mode_for_profiling_tests():`
put everything under this context manager. So autodiff will actually be triggered.